### ISSUE TITLE
Cannot use LOAD CSV in created search

### ISSUE THREAD

**Issue by lawsonvt** [neo4j-apps/neo4j-bloom#20]
Hi

I have been trying to find a way to restore a session in bloom, i.e. have some way for the user to pull up the graph they have created without having to manually recreate it. Since I saw no integrated way to do this (please correct me if I am wrong), I thought I could do so by creating a custom cypher search to load the exported CSV file. I placed the export file 'bloom-relationships-export.csv' in the import folder and the following code executed in the browser does what i want it to:

`LOAD CSV FROM 'file:///bloom-relationships-export.csv' AS line
MATCH (n1) - [r] - (n2) 
WHERE ID(r) = toInteger(line[0])
RETURN n1,n2,r`

However, when I make a custom search in bloom and use the exact same chunk of code, I get the following error:

`AN ERROR OCCURRED WHILE EXECUTING SEARCH:
Unexpected end of input: expected '\', ANY or ''' (line 4, column 32 (offset: 110))
"RETURN n1,n2,r SKIP 0 LIMIT 500"`

I assume additional code is being appended to the executed code chunk, could this be causing errors? Can this code even be run in bloom?

**Comment by lawsonvt** [comment]
So I think I have solved my own issue and am posting my solution here for future people to find.

After much experimenting, I converted my initial 'LOAD CSV' command into a single line of code (it might not be evident, but my previous code had multiple newlines). This made the error message more detailed as I could see the total command that bloom was actually executing.
So the command:
`LOAD CSV FROM 'file:///bloom-relationships-export.csv' AS line MATCH (n1) - [r] - (n2) WHERE ID(r) = toInteger(line[0]) RETURN n1,n2,r`

Gave the error:
`AN ERROR OCCURRED WHILE EXECUTING SEARCH: Unexpected end of input: expected '\', ANY or ''' (line 1, column 38 (offset: 37))
"LOAD CSV FROM 'file: SKIP 0 LIMIT 500"`

So this leads me to believe that something about the '///' is causing the remaining command to be stripped down to:
`LOAD CSV FROM 'file: SKIP 0 LIMIT 500`

HOWEVER, I was able to get this all to work using the `apoc.load.csv()` function. Before implementing this, be sure you have added the APOC plugin to your session and have added the line `apoc.import.file.enabled=true` to your neo4j.conf file. When you then export the csv file, place it in the import folder for your database and unzip it. Whatever you named the zip file will serve as input for this search.

Search phrase: Reload export $export_folder

Cypher Query:
`CALL apoc.load.csv($export_folder + "/bloom-relationships-export.csv")
YIELD lineNo, map, list
MATCH (n1) - [r] - (n2) 
WHERE ID(r) = apoc.convert.toInteger(list[0])
RETURN n1,n2,r`

If anyone has any improvements for this, feel free to let me know.

**Comment by jharris4** [comment]
Hi! Sorry for the delayed response, and thanks for doing the investigation on this issue.

I did some investigating in the Bloom code and you are correct about the `///` being the cause of the issue. Bloom has code that filters out comments starting with `//`.

I've raised an internal issue with the Bloom team to look at whether we can improve the comment filtering code to account for file paths with multiple slashes etc.
